server/test/unit/server/webservices/views/test_content.py

```diff
@@ -1,10 +1,484 @@
+from gettext import gettext as _
+import json
+import mock
 import unittest
 
-import mock
+from django.http import HttpResponseNotFound, HttpResponseRedirect
+
+from .base import assert_auth_DELETE, assert_auth_READ, assert_auth_CREATE, assert_auth_UPDATE
+from pulp.server.exceptions import InvalidValue, MissingResource, OperationPostponed
+from pulp.server.webservices.views.content import (
+    CatalogResourceView,
+    ContentTypeResourceView,
+    ContentTypesView,
+    ContentUnitResourceView,
+    ContentUnitsCollectionView,
+    DeleteOrphansActionView,
+    OrphanCollectionView,
+    OrphanTypeSubCollectionView,
+    OrphanResourceView,
+    UploadResourceView,
+    UploadsCollectionView,
+    UploadSegmentResourceView,
+)
+
+
+class TestOrphanCollectionView(unittest.TestCase):
+    """
+    Tests for views for all orphaned content.
+    """
+
+    @mock.patch('pulp.server.webservices.controllers.decorators._verify_auth',
+                new=assert_auth_READ())
+    @mock.patch('pulp.server.webservices.views.content.generate_json_response')
+    @mock.patch('pulp.server.webservices.views.content.factory')
+    def test_get_orphan_collection_view(self, mock_factory, mock_serializer):
```

Comment:
Is the variable name 'mock_serializer' the best choice? It seems to mock genereate_json_response? maybe 'mock_response'?